1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,53 @@
 function readingList(books) {
   // Write your code here...
+  let divContent = document.querySelector("#content");
+  let ulElement = document.createElement("ul");
+  divContent.appendChild(ulElement);
+
+  for (let book of books) {
+    let liElement = document.createElement("li");
+    let divInner = document.createElement("div");
+    divInner.style.width = "95%";
+    let pElement = document.createElement("p");
+    let pText = document.createTextNode(book.title + " by " + book.author);
+    pElement.appendChild(pText);
+    pElement.style.fontSize = "20px";
+    pElement.style.fontWeight = "500";
```

Comment:
how could you add these styles as a class to avoid adding multiple inline styles?

Reply:
I created p  tag in CSS and deleted these 2 lines.
p {
  font-size: 20px;
  font-weight: 500;
}